### PR TITLE
docs: add a debugging tip

### PR DESCRIPTION
**Pull request by yanivefraim**

### CODE REVIEW

**Comment by aslushnikov** [comment]
@ebidel: I remember there were some precautions to help rendering Puppeteer page on developers.google.com. I this patch fine?

File: README.md [hunk 1]
```diff
@@ -211,21 +211,21 @@ Puppeteer creates its own Chromium user profile which it **cleans up on every ru
 
 4. Stop test execution and use a debugger in browser
 
-  - Use `{devtools: true}` when launching Puppeteer:
+    - Use `{devtools: true}` when launching Puppeteer:
```
Comment (ebidel) [inline README.md]: Let's keep the indentation the same so there are no issues when we import the docs to developers.google.com.

**Comment by yanivefraim** [comment]
@ebidel - sure, fixed. Is it ok now?

**Review by ebidel — APPROVED** [review]

**Comment by aslushnikov** [comment]
@ak239 will love this.

**Pull request merged**